system.go

```diff
@@ -22,6 +22,81 @@ import (
+	InitiateShutdownFlagHybrid             InitiateShutdownFlag = wrappers.SHUTDOWN_HYBRID
+)
+
+type InitiateShutdownReason uint32
+
+const (
+	InitiateShutdownReasonUserDefinied InitiateShutdownReason = wrappers.SHTDN_REASON_FLAG_USER_DEFINED
+	InitiateShutdownReasonPlaned       InitiateShutdownReason = wrappers.SHTDN_REASON_FLAG_PLANNED
+)
+
+const (
+	InitiateShutdownReasonMajorOther          InitiateShutdownReason = wrappers.SHTDN_REASON_MAJOR_OTHER
+	InitiateShutdownReasonMajorHardware       InitiateShutdownReason = wrappers.SHTDN_REASON_MAJOR_HARDWARE
+	InitiateShutdownReasonMajorperatingSystem InitiateShutdownReason = wrappers.SHTDN_REASON_MAJOR_OPERATINGSYSTEM
+	InitiateShutdownReasonMajorSoftware       InitiateShutdownReason = wrappers.SHTDN_REASON_MAJOR_SOFTWARE
+	InitiateShutdownReasonMajorApplication    InitiateShutdownReason = wrappers.SHTDN_REASON_MAJOR_APPLICATION
+	InitiateShutdownReasonMajorSystem         InitiateShutdownReason = wrappers.SHTDN_REASON_MAJOR_SYSTEM
+	InitiateShutdownReasonMajorPower          InitiateShutdownReason = wrappers.SHTDN_REASON_MAJOR_POWER
+	InitiateShutdownReasonMajorLegacyApi      InitiateShutdownReason = wrappers.SHTDN_REASON_MAJOR_LEGACY_API
+)
+
+const (
+	InitiateShutdownReasonMinorOther                InitiateShutdownReason = wrappers.SHTDN_REASON_MINOR_OTHER
+	InitiateShutdownReasonMinorMaintenance          InitiateShutdownReason = wrappers.SHTDN_REASON_MINOR_MAINTENANCE
+	InitiateShutdownReasonMinorInstallation         InitiateShutdownReason = wrappers.SHTDN_REASON_MINOR_INSTALLATION
+	InitiateShutdownReasonMinorUpgrade              InitiateShutdownReason = wrappers.SHTDN_REASON_MINOR_UPGRADE
+	InitiateShutdownReasonMinorReconfig             InitiateShutdownReason = wrappers.SHTDN_REASON_MINOR_RECONFIG
+	InitiateShutdownReasonMinorHung                 InitiateShutdownReason = wrappers.SHTDN_REASON_MINOR_HUNG
+	InitiateShutdownReasonMinorUnstable             InitiateShutdownReason = wrappers.SHTDN_REASON_MINOR_UNSTABLE
+	InitiateShutdownReasonMinorDisk                 InitiateShutdownReason = wrappers.SHTDN_REASON_MINOR_DISK
+	InitiateShutdownReasonMinorProcessor            InitiateShutdownReason = wrappers.SHTDN_REASON_MINOR_PROCESSOR
+	InitiateShutdownReasonMinorNetwordCard          InitiateShutdownReason = wrappers.SHTDN_REASON_MINOR_NETWORKCARD
+	InitiateShutdownReasonMinorPowerSupply          InitiateShutdownReason = wrappers.SHTDN_REASON_MINOR_POWER_SUPPLY
+	InitiateShutdownReasonMinorCordunplugged        InitiateShutdownReason = wrappers.SHTDN_REASON_MINOR_CORDUNPLUGGED
+	InitiateShutdownReasonMinorEnvironment          InitiateShutdownReason = wrappers.SHTDN_REASON_MINOR_ENVIRONMENT
+	InitiateShutdownReasonMinorHardwareDriver       InitiateShutdownReason = wrappers.SHTDN_REASON_MINOR_HARDWARE_DRIVER
+	InitiateShutdownReasonMinorOtherdriver          InitiateShutdownReason = wrappers.SHTDN_REASON_MINOR_OTHERDRIVER

```

Comment:
[Minor] `Otherdriver` -> `OtherDriver`.

Similarly, `Bluescreen` -> `BlueScreen`, `Securityfix` -> `SecurityFix`, etc. etc.

Reply:
ok. I did the capitalization in a similar way to the underscores in MS constansts. But you right it looks bad.